### ISSUE TITLE
Use tag forge:bolts/copper instead of hardcoded COPPER_BOLT

### ISSUE THREAD

**Issue by Exzept1on** [HyperDashPony/FirmaCiv#33]
Here: https://github.com/HyperDashPony/FirmaCiv/blob/81b22b5a0f56871240d9ab0bcfcba67727d398d7/src/main/java/com/alekiponi/firmaciv/common/block/WoodenBoatFrameBlock.java#L60C18-L60C18

Here: https://github.com/HyperDashPony/FirmaCiv/blob/81b22b5a0f56871240d9ab0bcfcba67727d398d7/src/main/java/com/alekiponi/firmaciv/common/block/WoodenBoatFrameBlock.java#L60C18-L60C18

This helps modpack devs create own bolts or something like this. Because now I should use weird mixins to fix that

**Comment by HyperDashPony** [comment]
I'll fix this in 0.2.0

**Comment by HyperDashPony** [comment]
Actually, since this isn't a block entity, it's not possible to do that since there's no ItemStack storing the item. It has to be explicit, it can't be a tag.

**Comment by Traister101** [comment]
I'll pop in here since I wrote that stuff. We can't really use a tag. Our bolt item is checked for explicitly to add progress to the frame as well as explicitly returned in the loot table/when right clicked.

Originally we had a light weight block entity used for storing the planks and bolts but I noticed that was actually redundant as we needed the blockstates for the changes to the model. We could of course put a block entity back but it feels super unnecessary simply to support "dynamic" bolt items. And for another thing the bolt textures are always going to be copper unless you change it in a resource pack so, it really just doesn't make much sense imho.

Modpack devs shouldn't really be making their own bolt items for frames as they fundamentally don't make sense anyway.
